toolchain/parse/state.def

```diff
@@ -402,7 +402,7 @@ CARBON_PARSE_STATE(Expr)
 //
 //  (
 // ^
-//   1. ParenExpr
+//   1. OpenParen
```

Comment:
I don't see a definition of the `OpenParen` state here. Should this be `ParenExpr`?